### DISCUSSION TITLE
Decide on M setup

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#69] [General]
@okenk , Id like to decide which M setup to use as the base setup moving forward for future model explorations. Ive added some pros and cons to each. Please edit this post to add in any you see. In the comments we can discuss preferences

**Age-invarying**
*Pros*: Seems more biologically reasonable 
*Cons*: Limited support - not being estimated to be different

**Ramp**
*Pros*: Consistent with previous assessments; biological basis (around maturity) to explain, in between other two
*Cons*: A ramp in M does not have a great biological explanation, not extreme enough to fit sex ratio data

**Breakpoint**
*Pros*: A single shift has slightly better biological rationale than a ramp, better match with sex ratio in data, reduces early rec dev pattern somewhat
*Cons*: Value is not based in biology but rather data, extreme assumption, M of ~0.15 would assume females max age is 36 and we see them larger

**Comment by brianlangseth-NOAA** [comment]
Im leaning towards the ramp primarily because M ~ 0.15 (which translates to longevity of 36) seems pretty extreme. This option also is a mid point between assumptions (breakpoint and constant). It doesn't fit the data as well as the breakpoint (which also resolves some of the early recdev issue) but recent data has more of a 50-50 ratio.

**Comment by okenk** [comment]
I am sitting next to Owen. I showed him these three options, the survey data (but the survey does not go into the rocks), and the fishery data from the ODFW study (where sex ratio is -> 50/50 in the last decade, but is confounded with fishing dynamics).

He actually suggested the constant M, since that is more consistent with how we model rockfish M broadly, and increasing M with age is highly unusual. He also suggested pursuing multiple. Breakpoint brackets better, but he agrees constant M and ramp M are more plausible.

**Reply by brianlangseth-NOAA** [reply]
Would be interesting to hear why he thinks the constant model doesn't estimate something different for female. The model clearly wants to have fewer old fish, but not at the expense of killing off smaller fish. 

I imagine if steepness were higher (potential for more little ones) M may creep up more in the constant scenario. 

Ideas to possibly explore include 'profiling' over the breakpoint value, and over steepness

**Comment by okenk** [comment]
We did not look extensively at the fits. It looks like recently there is a similar proportion of males and females across all ages in the Oregon trawl fishery. So why should it estimate different M's? It might be worth comparing likelihoods or fits to comp data.

**Reply by brianlangseth-NOAA** [reply]
I looked at fits - and both age and length comps improve (lower likelihood) as one progresses from constant - ramp - breakpoint. Sex ratios move towards more females as well, I dont know how to quantify the 'fits' to sex_ratios

**Comment by okenk** [comment]
Do we have sex-specific selectivity?

**Reply by brianlangseth-NOAA** [reply]
No

**Reply by okenk** [reply]
I am spitballing here, but I wonder if more old females are on the rocks than older males, so the survey selectivity will be more domed for females than males. But maybe I am off!

**Comment by brianlangseth-NOAA** [comment]
Ideas to explore:

- [ ] 'profile' over a breakpoint ages, say 6-24 by 2
~- [ ] profile over steepness~ I estimated steepness in 4_0_8 and M for female didn't change. I dont think a profile will be fruitful to diagnose right now - though we will need to do it. We can do it later. 
- [ ] sex-specific selectivity
    - [ ] Could try the descending limb age-selectivity idea from Rick
    - [x] Try estimating just the peak selectivity for females and keep all other parameters the same
    - [x] Probably will also need to estimated separate selex all together (though this doesn't make much biological sense because no fleet is catching them)

**Reply by brianlangseth-NOAA** [reply]
Copied over remaining ideas into issue #67. Can close this
